test/dummy/app/components/dummy/mailer/preview_text_component.slim

```diff
@@ -0,0 +1,11 @@
+.d-mailer-preview-text
+  / Preview text must be at least 90 characters long
+  = preview_text
+
+/ Hack for not adding other distracting text or characters
+.d-mailer-preview-text__hack
+  == "&#847; &zwnj; &nbsp; &#8199; &shy; &#847; &zwnj; &nbsp; &#8199; &shy; " \
+     "&#847; &zwnj; &nbsp; &#8199; &shy; &#847; &zwnj; &nbsp; &#8199; &shy; " \
+     "&#847; &zwnj; &nbsp; &#8199; &shy; &#847; &zwnj; &nbsp; &#8199; &shy; " \
+     "&#847; &zwnj; &nbsp; &#8199; &shy; &#847; &zwnj; &nbsp; &#8199; &shy; " \
+     "&#847; &zwnj; &nbsp; &#8199; &shy; &#847; &zwnj; &nbsp; &#8199; &shy;"
```

Comment:
Jaky je vyznam teto komponenty? Proc je v mailu ten vychozi text?

Reply:
Ten vychozi text byla blbost, uz jsem to predelal

Tahle komponenta je pro zobrazeni "nahledoveho textu" (text vedle/pod předmětem emailu v inboxu). Lze takhle jednoduse nastavit, co se bude v nahledovem textu zobrazovat. Bez nahledove textu se vetsinou automaticky bere prvni text v newsletteru. Nekteri klienti ale misto toho berou alt text z obrazku. Nekteri dokonce zahrnuji html znacky

Ta cast s HTML entitami je tam kvuli tomu, aby za nahledovym textem uz nic nebylo a nepridaval se do nej nevyzadany text

Reply:
Je to hack, o kterem psali na Litmusu - https://www.litmus.com/blog/the-little-known-preview-text-hack-you-may-want-to-use-in-every-email